opencti-platform/opencti-front/src/private/components/drafts/DraftContextBanner.tsx

```diff
@@ -228,7 +229,7 @@ const DraftContextBannerComponent: FunctionComponent<DraftContextBannerComponent
             {t_i18n('Exit draft')}
           </Button>
         </div>
-        {currentAccessRight.canEdit && (
+        {useGranted([KNOWLEDGE_KNUPDATE_KNDELETE]) && currentAccessRight.canEdit && (
```

Comment:
valid, but not very fan it's very fragile since hook call is hidden inside the JSX and it's very easy to add condition before and not seeing that it's break the hook behind